### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_New Connection_ScreenReader: Screen reader JAWS is not announcing the information under hostname edit on navigating to hostname.

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17617]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTag:#A11yTCS; #Win11; #DesktopApp; #BM_mssql for VSCode_Win32_Mar2023; #A11ySev2; #Benchmark; #mssql for VSCode; #HighContrast; #503.2;

**Environment Details:**
Application Name: mssql for VSCode
Operating System: Win11[22H2]
Visual studio code insider: 1.77.0-insider (user setup)
Extension Version: v1.17.1

**Repro Steps:**
1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button 
5. Turn on the screen reader JAWS 
6. Tab till "Hostname" edit 

**Actual:**
Screen reader JAWS is not announcing the information under hostname edit on navigating to hostname.
**Note:** issue is not repro with NVDA Screen reader.

**Expected:**
 JAWS Screen reader should announce the information which is under "hostname" as that information is useful.

**User Impact:**
Screen reader users will miss the information under "hostname" as a result of it they cannot follow the information provided.

**Attachment**

**Comment by kburtram** [comment]
This UI has been removed from the product and is no longer applicable.
